plugins/basic/nexus-repository-nuget/src/main/java/com/sonatype/nexus/repository/nuget/internal/security/NugetApiKeyStoreImpl.java

```diff
@@ -62,6 +62,12 @@ public NugetApiKeyStoreImpl(final @Named("security") Provider<DatabaseInstance>
     this.databaseInstance = checkNotNull(databaseInstance);
     this.entityAdapter = checkNotNull(entityAdapter);
     this.principalsHelper = checkNotNull(principalsHelper);
+    try {
```

Comment:
need a place to start this from, @all is there a common paradigm for initialization or simply EventBus+NexusStartedEvent on a lifecycle component to bootstrap this?